loky/backend/context.py

```diff
@@ -141,6 +148,16 @@ def Event(self):
             return Event()
 
 
+class LokyNoMainContext(LokyContext):
+    def Process(self, *args, **kwargs):
+        kwargs.pop('init_main_module', False)
+        return LokyProcess(*args, init_main_module=False, **kwargs)
```

Comment:
New backend which enforces `init_main_module = False`. This is for specific use in `joblib` to allow usage of cloudpickle in script like files.

I will add a docstring.